sdk/database/dbplugin/v5/grpc_database_plugin.go

```diff
@@ -44,6 +45,12 @@ func (d GRPCDatabasePlugin) GRPCServer(_ *plugin.GRPCBroker, s *grpc.Server) err
 			factoryFunc: d.FactoryFunc,
 			instances:   make(map[string]Database),
 		}
+
+		// Multiplexing is enabled for this plugin, register the server so we
+		// can tell the client in Vault.
+		pluginutil.RegisterPluginMultiplexingServer(s, pluginutil.PluginMultiplexingServerImpl{
+			Supported: true,
```

Comment:
Do you think it'd be useful to also register a `PluginMultiplexingServer` with `Supported` set to false on the case above to be able to differentiate between `Unimplemented` and an explicit `false`? That we we can detect the difference between a plugin that can register such server (i.e. an older plugin) vs one that choses not to.